src/Broadcast.js

```diff
@@ -137,6 +137,16 @@ class Broadcast {
     out = out.join('\r\n').replace('<NEWLINE>', '\r\n');
     return out;
   }
+
+  static pad(pad, str, padLeft) {
```

Comment:
This is a slightly strange API for a pad function. What's more, we already have a `left-pad` package imported that you can use. https://www.npmjs.com/package/left-pad